src/main/scripts/org/reaktivity/specification/nukleus/tcp/control/route/client.host.and.subnet/nukleus.rpt

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nukleiRG010101N ${nuklei:directory("target/nukleus-itests").controlCapacity(1024, 1024)}
+
+property controlRG010101N ${nukleiRG010101N.controlNew("tcp")}
+
+connect "agrona://stream/bidirectional"
+        option agrona:reader ${agrona:manyToOneReader(controlRG010101N.nukleus)}
+        option agrona:writer ${agrona:broadcastTransmitter(controlRG010101N.controller)}
+
+connected
+
+read 0x00000001
+read (long:correlationIdRG010101N)
+read [0x01]
+read [0x06] "source"
+read 0L
+read [0x0a] "google.com"
+read (long:localhostPortRefRG010101N)
+read [0..8] # authorization
+
+write 0x40000001
+write ${correlationIdRG010101N}
+write ${nuklei:newReferenceId()}
+write flush
+
+read 0x00000001
+read (long:correlationIdRG010101N2)
+read [0x01]
+read [0x06] "source"
+read 0L
```

Comment:
You'll need to read this (source ref) into a variable and then rounded out in the response